browser/src/react-redux/App/ForgotPasswordPage/ForgotPasswordPage.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const initialState = {
+  email: '',
+  response: '',
```

Comment:
I think it would be easy to confuse the word `response` with the response object. Let's stick to our current convention and go with something like `successMessage`.